webui/src/pages/repositories/repository/objects.jsx

```diff
@@ -37,16 +37,25 @@ import {
 import {Box} from "@mui/material";
 import {RepoError} from "./error";
 import { getContentType, getFileExtension, FileContents } from "./objectViewer";
+import {OverlayTrigger} from "react-bootstrap";
+import Tooltip from "react-bootstrap/Tooltip";
 
 
 const README_FILE_NAME = 'README.md';
 const REPOSITORY_AGE_BEFORE_GC = 14;
 
-const ImportButton = ({variant = "success", enabled = false, onClick}) => {
+const ImportButton = ({variant = "success", onClick, config }) => {
+    let tip = config.import_support ? "Import data from a remote source" :
+        "Unsupported for " + config.blockstore_type +" blockstore";
```

Comment:
```suggestion
        "Import is not supported when lakeFS is using " + config.blockstore_type +" blockstore.";
```

Can we also link to https://docs.lakefs.io/howto/import.html#limitations-to-importing-data, perhaps hyperlink `not supported` to it in the message

However… reading that page it sounds like it's not that local is not supported but that it has to be enabled. 

So perhaps the error should be not *enabled* rather than *supported* ? 

> Import is [not enabled](https://docs.lakefs.io/howto/import.html#limitations-to-importing-data) by default for local blockstore.

Reply:
See #5475 for a proposed amendment to clarify the doc too -- and then you could link to `import.html#importing-from-local-storage` from this message.

Reply:
I agree it should be more accurate. However it's a tooltip, so any message should be short.

Reply:
Can we just have an exception for `local` blockstore and have the message say "not enabled" instead of "not supported"

Reply:
Tooltips fade away when the mouse is not hovering the button, so I think linking there isn't the best choice.